src/libraries/System.Security.Permissions/src/CompatibilitySuppressions.xml

```diff
@@ -29,29 +21,29 @@
   <Suppression>
     <DiagnosticId>CP0001</DiagnosticId>
     <Target>T:System.Xaml.Permissions.XamlLoadPermission</Target>
-    <Left>ref/netcoreapp3.0/System.Security.Permissions.dll</Left>
+    <Left>ref/net5.0/System.Security.Permissions.dll</Left>
     <Right>lib/netstandard2.0/System.Security.Permissions.dll</Right>
```

Comment:
@Anipik why are we comparing a ref/5.0 asset from the previous package against the lib/netstandard2.0 asset on the new package if the new package contains a lib/net5.0... there is no way the netstandard2.0 asset would be chosen if the tfm is net5.0?